File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.BusinessLayer/RecepLista.cs
using LogisticStorage.DataLayer;
using LogisticStorage.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.BusinessLayer
{
    public class RecepLista
    {
        public static List<RecepListaEntity> ObtenerItem(Int32 ListaId)
        {
            RecepListaDB DB = new RecepListaDB();
            return DB.ObtenerItem(ListaId);
        }

        public static List<RecepListaEntity> ObtenerItems(String Codigo)
        {
            RecepListaDB DB = new RecepListaDB();
            return DB.ObtenerItems(Codigo);
        }
    }
}

File: ProyectoMysql/server/BusinessLayer/RecepLista.py
from DataLayer.RecepListaDB import *
from Utilidades.Conexion.configMysql import StartTransaction,EndTransaction


class RecepLista:

    def ObtenerItems(Codigo: str):
        try:
            data = RecepListaDB.ObtenerItems(Codigo)
            return data
        except Exception as e:
            print(e)
    
    def ObtenerItem(Id: int):
        try:
            data = RecepListaDB.ObtenerItem(Id)
            return data
        except Exception as e:
            print(e)